### ISSUE TITLE
why this code has not lr_scheduler.step()?

### ISSUE THREAD

**Issue by Gi-gigi** [SConsul/Global_Convolutional_Network#5]
The author commented out the update of the learning rate, why?
thank u sir
